AppCenterDataStorage/AppCenterDataStorage/Internal/Model/MSPaginatedDocumentsInternal.h

```diff
@@ -4,15 +4,19 @@
 #import "MSPage.h"
 #import "MSReadOptions.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface MSPaginatedDocuments ()
 
 // Read-only.
-@property(nonatomic, copy, readonly) NSString *partition;
-@property(nonatomic, readonly) Class documentType;
-@property(nonatomic, readonly) MSReadOptions *readOptions;
+@property(nonatomic, copy, readonly, nullable) NSString *partition;
```

Comment:
This should be non nullable. The change will need to be propagated.
```suggestion
@property(nonatomic, copy, readonly) NSString *partition;
```

Reply:
I see MSPaginatedDocuments class has a contrstuer which will pass nil value to these two properties, so I set them as nullable.
![image](https://user-images.githubusercontent.com/47266368/56334303-62c6e200-61ca-11e9-99e1-c369e8b061bc.png)
For your comment, I should pass non null value from them, right?